test/versioned/express/package.json

```diff
@@ -32,7 +40,34 @@
         "segments.tap.js",
         "transaction-naming.tap.js"
       ]
+    },
+    {
+      "engines": {
+        "node": ">=18"
+      },
+      "dependencies": {
+        "express": "5.0.0-beta.3",
+        "ejs": "2.5.9"
+      },
+      "files": [
```

Comment:
express-enrouten has been omitted because it doesn't work with express 5. the package hasn't had updates in 7 years and is marked as archived as well